src/flavors/__tests__/__snapshots__/openapi.test.ts.snap

```diff
@@ -100,6 +100,49 @@ Object {
     "schemas": Object {
       "ExampleDTO": Object {
         "properties": Object {
+          "nullableBooleanValue": Object {
+            "nullable": true,
+            "type": "boolean",
+          },
+          "nullableDateStringValue": Object {
+            "format": "date",
+            "nullable": true,
+            "type": "string",
+          },
+          "nullableDateValue": Object {
+            "format": "date-time",
+            "nullable": true,
+            "type": "string",
+          },
+          "nullableEnumValue": Object {
+            "$ref": "#/components/schemas/Object",
+            "nullable": true,
+          },
+          "nullableIntegerValue": Object {
+            "nullable": true,
+            "type": "integer",
+          },
+          "nullableNestedValue": Object {
```

Comment:
I am somewhat surprised that a nullable type reference gets defined using `allOf`, but this does seem to be valid (and maybe it's the only way?)